jujugui/static/gui/src/app/extensions/app-renderer-extension.js

```diff
@@ -37,43 +37,38 @@ YUI.add('app-renderer-extension', function(Y) {
 
       @method _renderBreadcrumb
       @param {Object} options
-        showEnvSwitcher: false
+        showEnvSwitcher: true
     */
     _renderBreadcrumb: function({ showEnvSwitcher=true } = {}) {
-      // If this.env is undefined then do not render the switcher because there
-      // is no env to connect to. It will be undefined when the breadcrumb
-      // is rendered in the callback for generateSocketUrl because an env
-      // has not yet been created.
-      var env = this.env;
-      var jem = this.jem;
-      var utils = views.utils;
-      // If gisf is enabled then we won't be connected to a model to know
-      // what facades are supported but we can reliably assume it'll be Juju 2
-      // or higher which will support the necessary API calls.
-      if (!this.get('gisf')) {
-        if(!env || env.findFacadeVersion('ModelManager') === null &&
-           env.findFacadeVersion('EnvironmentManager') === null) {
-          // We do not want to show the model switcher if it isn't supported as
-          // it throws an error in the browser console and confuses the user
-          // as it's visible but not functional.
-          showEnvSwitcher = false;
-        }
+      const env = this.env;
+      const controller = this.controllerAPI;
```

Comment:
Keeping calling this controllerAPI would be beneficial for future readers I believe.